### PR TITLE
Git man stack patch 1

### PR DESCRIPTION
**Pull request by gitMan-stack**
Co-authored-by: shinevue <shine401020@gmail.com>

### CODE REVIEW

**Review by shinevue — APPROVED** [review]
You welcome.

File: README.md [hunk 1]
```diff
@@ -60,4 +60,5 @@
 #### [Achilles](https://github.com/oleh1010)
 #### [Chivalrousdev](https://github.com/chivalrousdev)
 #### [ScottKing](https://github.com/creative2113)
+#### [Virus](https://github.com/gitMan-stack)
```
Comment (BlackGhost2693) [inline README.md:64]: #### [Blackghost2693](https://github.com/blackghost2693)

Comment (shinevue) [reply README.md:64]: Add review

**Review by creative2113 — APPROVED** [review]
please go ahead

**Review by BlackGhost2693 — APPROVED** [review]
Welcome

**Review by joellake992 — COMMENTED** [review]
Congratulation

**Review by Johnhvy — APPROVED** [review]
Okay
I agree

**Review by techietrend — APPROVED** [review]
Let's merge now

**Review by joellake992 — APPROVED** [review]
Let's merge now

**Comment by gitMan-stack** [comment]
Let's start  new project

**Pull request merged**